src/Forms/src/Xamarin.Forms.Core/Region.cs

```diff
@@ -10,7 +10,7 @@ public struct Region
 		// such as circles if required, without affecting anything else.
 
 		IReadOnlyList<Rectangle> Regions { get; }
-		readonly Thickness _inflation;
+		readonly Thickness? _inflation;
```

Comment:
this code was compiling with an error because inside this class it does null checks but the type can't be null

so I made the type nullabel